net-gateway/src/endpoints/charts/network_graph/request/manager.rs

```diff
@@ -38,21 +37,22 @@ impl ChartRequestManagaer for NetworkGraphChartManager {
 
     fn form_dto_request(
         &self,
-        params: Arc<web::Query<GeneralFilters>>,
+        params: Arc<GeneralFilters>,
         #[allow(unused_variables)]
-        client_data: Arc<web::Query<ClientData>>
+        client_data: Arc<ClientData>,
+        filters: Option<Arc<Filters>>,
     ) -> Box<dyn API> {
         Box::new(NetworkGraphRequestDTO::new(
             params.start_date,
             params.end_date,
-            false
+            filters.as_ref().unwrap().as_ref().clone().into(),
```

Comment:
IDK, just a matter of codestyle (as for compiler this code will remain the same), but I prefer have such long line as its one variable instead.